sql/build.gradle

```diff
@@ -17,6 +17,8 @@ dependencies {
     compile project(':sql-parser')
     compile 'com.amazonaws:aws-java-sdk-s3:1.10.33'
     compile 'org.apache.commons:commons-math3:3.4.1'
+    compile 'org.apache.commons:commons-lang3:3.3.2'
```

Comment:
please remove the dependencies.. the usages of lang3 can be replaced with `Booleans` from guava